chart/otomi/templates/job.yaml

```diff
@@ -54,10 +52,8 @@ spec:
             - name: values-secret
               mountPath: /secret
         - name: map-values
-          image: {{ .Values.tasksImage.repository }}:{{ .Values.tasksImage.tag | default "latest" }}
-          {{- with .Values.image.pullPolicy }}
-          imagePullPolicy: {{ . }}
-          {{- end }}
+          image: otomi/tasks:latest
+          imagePullPolicy: Always
```

Comment:
please use this logic to determine which to choose:

if starts with 'v' > `IfNotExists`
otherwise `Always`

Reply:
what if a dev branch starts with 'v'?

Reply:
then check if char(2) is a number?

Reply:
or maybe simply let devs to provide pullPolicy and KISS the logic?

Reply:
nah, this is about our code...so we can avoid this, especially since the tag is coming from another place already

Reply:
It is a very easy piece of logic...then we can close this @mojtabaimani